src/components/toast/index.js

```diff
@@ -172,7 +173,7 @@ export default class Toast extends BaseComponent {
   }
 
   render() {
-    const {backgroundColor, actions, allowDismiss } = this.getThemeProps();
+    const {backgroundColor, actions, allowDismiss, blur} = this.getThemeProps();
```

Comment:
blur is missing the component propTypes. 
also to prefer maybe calling it blurOptions to be more explicit